### PR TITLE
DOC: Docstring of trait model

### PR DESCRIPTION
**Pull request by daikitag**
Update the docstring of trait model, as it will be used in the documentation to explain the corresponding model.

### CODE REVIEW

File: tstrait/trait_model.py [hunk 1]
```diff
@@ -144,6 +144,10 @@ class TraitModelExponential(TraitModel):
     so please see its documentation for the details of the exponential distribution
     simulation.
 
+    If `negative` is False, only positive values will be simulated as part of the
```
Comment (jeromekelleher) [inline tstrait/trait_model.py:147]: We decided to call it ``random_sign``, didn't we?

**Review by jeromekelleher — COMMENTED** [review]
Looks like you're mixing up the name of the parameter here. Is it ``negative`` or ``random_sign``?

**Comment by daikitag** [comment]
> Looks like you're mixing up the name of the parameter here. Is it `negative` or `random_sign`?

Thank you very much for spotting that. I will fix it now.

File: tstrait/trait_model.py [hunk 2]
```diff
@@ -122,10 +122,11 @@ class TraitModelExponential(TraitModel):
     ----------
     scale : float
         Scale of the exponential distribution. Must be non-negative.
-    negative : bool, default False
-        Determine if a negative can be simulated from the trait model. If it is set
-        to be True, :math:`1` or :math:`-1` will be randomly multiplied to the
-        simulated effect sizes.
+    random_sign : bool, default False
+        If True, :math:`1` or :math:`-1` will be randomly multiplied to the
+        simulated effect sizes, such that we can simulate constant value effect
```
Comment (jeromekelleher) [inline tstrait/trait_model.py:127]: this "constant value" is incorrect right? Can you be a bit more careful about copying and pasting text around please?

**Review by jeromekelleher — COMMENTED** [review]
Looks good, but I think you've copy-pasted a description from the fixed effect model incorrectly

File: tstrait/trait_model.py (hunk 2 above)
Comment (daikitag) [reply tstrait/trait_model.py]: I'm sorry about that, and I just fixed it now.

**Review by jeromekelleher — APPROVED** [review]

**Pull request merged**